.gitlab-ci.yml

```diff
@@ -8,21 +8,22 @@ stages:
 
 variables:
   DOCKER_REPOSITORY: mendersoftware/gui
+  NODE_IMAGE_VERSION: 14.7.0-alpine
```

Comment:
I suggest setting `image: node:14.7.0-alpine` above instead (and adding `image: docker:git` to `publish:disclaimer`). The node image is used in all jobs except acceptance testing and the mentioned disclaimer publish, so it makes more sense to set it as the "standard".

Reply:
A very good point, thanks! changed it as suggested - I guess I got blinded 🙈